### PR TITLE
docs: add forward-auth zh document

### PR DESCRIPTION
**Pull request by biubiue**
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by zaunist** [comment]
Hi, @biubiue . Please check the doc style.

File: docs/zh/latest/plugins/forward-auth.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+3. **client_headers** 当授权失败时转发 `authorization` 服务响应头到 `client`
+
+```shell
+$ curl -i http://127.0.0.1:9080/headers
+HTTP/1.1 403 Forbidden
+Location: http://example.com/auth
+```
+
+最后,你可以通过在路由中移除的方式禁用`forward-auth`插件。
```
Comment (zaunist) [inline docs/zh/latest/plugins/forward-auth.md:140]:
```suggestion
最后,你可以通过在路由中移除的方式禁用 `forward-auth` 插件。
```
You can ref this repo https://github.com/sparanoid/chinese-copywriting-guidelines

Comment (biubiue) [reply docs/zh/latest/plugins/forward-auth.md:140]: ok

File: docs/zh/latest/plugins/forward-auth.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+## 数据定义
+
+request_headers属性中转发到 `authorization` 服务中的APISIX内容清单
+| Scheme | HTTP Method | Host | URI | Source IP |
+| -- | -- | -- | -- | -- |
+| X-Forwarded-Proto | X-Forwarded-Method | X-Forwarded-Host | X-Forwarded-Uri | X-Forwarded-For |
+
+## 示例
+
+首先, 你需要设置一个认证服务。这里使用的是APISIX无服务器插件模拟的示例。
```
Comment (zaunist) [inline docs/zh/latest/plugins/forward-auth.md:62]:
```suggestion
首先, 你需要设置一个认证服务。这里使用的是 Apache APISIX 无服务器插件模拟的示例。
```

File: docs/zh/latest/plugins/forward-auth.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            "phase": "rewrite",
+            "functions": [
+                "return function (conf, ctx) local core = require(\"apisix.core\"); local authorization = core.request.header(ctx, \"Authorization\"); if authorization == \"123\" then core.response.exit(200); elseif authorization == \"321\" then core.response.set_header(\"X-User-ID\", \"i-am-user\"); core.response.exit(200); else core.response.set_header(\"Location\", \"http://example.com/auth\"); core.response.exit(403); end end"
+            ]
+        }
+    }
+}'
+```
+
+下一步, 我们创建一个测试路由.
```
Comment (zaunist) [inline docs/zh/latest/plugins/forward-auth.md:81]:
```suggestion
下一步, 我们创建一个测试路由。
```

File: docs/zh/latest/plugins/forward-auth.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+            "httpbin.org:80": 1
+        },
+        "type": "roundrobin"
+    }
+}'
+```
+
+我们可以进行下面三个测试：
+
+1. **request_headers** 从 `client`转发请求头到 `authorization` 服务
```
Comment (zaunist) [inline docs/zh/latest/plugins/forward-auth.md:107]:
```suggestion
1. **request_headers** 从 `client` 转发请求头到 `authorization` 服务
```

File: docs/zh/latest/plugins/forward-auth.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+        "X-User-ID": "i-am-user",
+        "Next": "More-headers"
+    }
+}
+```
+
+3. **client_headers** 当授权失败时转发 `authorization` 服务响应头到 `client`
+
+```shell
+$ curl -i http://127.0.0.1:9080/headers
```
Comment (zaunist) [inline docs/zh/latest/plugins/forward-auth.md:135]:
```suggestion
curl -i http://127.0.0.1:9080/headers
```

File: docs/zh/latest/plugins/forward-auth.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+            ]
+        }
+    }
+}'
+```
+
+下一步, 我们创建一个测试路由.
+
+```shell
+$ curl -X PUT http://127.0.0.1:9080/apisix/admin/routes/1
```
Comment (zaunist) [inline docs/zh/latest/plugins/forward-auth.md:84]:
```suggestion
curl -X PUT http://127.0.0.1:9080/apisix/admin/routes/1
```

File: docs/zh/latest/plugins/forward-auth.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+| Scheme | HTTP Method | Host | URI | Source IP |
+| -- | -- | -- | -- | -- |
+| X-Forwarded-Proto | X-Forwarded-Method | X-Forwarded-Host | X-Forwarded-Uri | X-Forwarded-For |
+
+## 示例
+
+首先, 你需要设置一个认证服务。这里使用的是APISIX无服务器插件模拟的示例。
+
+```shell
+$ curl -X PUT 'http://127.0.0.1:9080/apisix/admin/routes/auth' \
```
Comment (zaunist) [inline docs/zh/latest/plugins/forward-auth.md:65]:
```suggestion
curl -X PUT 'http://127.0.0.1:9080/apisix/admin/routes/auth' \
```
Maybe remove `$` is better ?

Comment (biubiue) [reply docs/zh/latest/plugins/forward-auth.md:65]: ok

**Pull request closed without merging**